tasks/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -313,12 +321,17 @@ spec:
             infra_pr = github_app.create_mr()
             if "url" not in infra_pr:
                 infra_pr = github_app.get_pr()
-            description = infra_pr["body"]
-            if description == None:
-                description = "Included PRs:"
-            new_pr_link = github_app.get_pr_url_from_sha(revision)
-            new_description = f"{description}\r\n- {new_pr_link}"
-            github_app.update_mr_description(infra_pr["url"], new_description)
+            if "body" in infra_pr:
+                description = infra_pr["body"]
+                if description == None:
+                    description = "Included PRs:"
+                new_pr_link = github_app.get_pr_url_from_sha(revision)
+                new_description = f"{description}\r\n- {new_pr_link}"
```

Comment:
What's the `\r` in the string for? That's carriage return without new line, right? But it's followed by a new line, so it can be just removed?

Reply:
not sure. this was taken from the tekton-ci pipeline. I can change if you prefer.

Reply:
OK, probably not worth bothering right now.